### DISCUSSION TITLE
VUCC statistics at home page

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1575] [Ideas / Feature Requests]
Hi Peter,

Similar to "Countries Breakdown" on home page, is it possible to have a VUCC info there?
(maybe as optional -- DXCC, VUCC, QSLs, QSOs...)

**Comment by magicbug** [comment]
Its possible but they are "expensive" SQL calls as in take time to generate, unless perhaps these are cached and stored every day so its not instant but a 24 delay

**Comment by 4O4A** [comment]
I understand. As far as I am concerned, I will be fine with once a day update. Just like to see all interesting numbers on one place :)
Also, I will fine if it remains as is. :)

**Comment by magicbug** [comment]
No its a reasonable suggestion

**Comment by AndreasK79** [comment]
Just for the fun of it, I added it myself:
![image](https://user-images.githubusercontent.com/6977712/188286806-e1da2739-99c0-4a4b-baa6-3f55eae95899.png)
Page loading didn't increase too much either:
With:
![image](https://user-images.githubusercontent.com/6977712/188286819-5b91654f-896d-4505-86a8-569e903e2638.png)
Without:
![image](https://user-images.githubusercontent.com/6977712/188286823-0a8dd14c-01a8-455a-8550-9bff6182137f.png)

Depending on what information should be shown, my test shows that it can probably be done without slowing down the front page that much.

**Comment by AndreasK79** [comment]
This is now merged into the station_logbooks branch.
